apps/ftu/js/wifi.js

```diff
@@ -193,35 +193,47 @@ var WifiUI = {
     window.history.back();
   },
 
-  connect: function wui_connect(ssid, password, user) {
-    // First we check if there is a previous selected network
-    // and we remove their status
-    var networkSelected = document.querySelector('li[data-wifi-selected]');
-    if (networkSelected) {
-      var icon = networkSelected.querySelector('aside');
-      networkSelected.removeAttribute('data-wifi-selected');
-      networkSelected.classList.remove('connected');
+  _selectNetworkItem: function(ssid, isSelected=true) {
```

Comment:
:_) so happy to see default parameters.

I was about to ask if the linter accepts them, but I guess this PR is the proof :)